Services/userMetricService.js

```diff
@@ -0,0 +1,65 @@
+let model = require("../Models/metrics/user").user;
+
+let metricDB = require("knex")({
+  client: "pg",
+  connection: process.env.DATABASE_URL
+});
+
+async function updateUserLoginTime(username){
+  let checkExists = null;
+  try {
+    await metricDB("users").where({
+      username: username
+    }).update({
+      lastlogin: metricDB.fn.now()
+    }).returning("*").then(res => {
+      checkExists = res;
+      console.log(checkExists);
+    });
+
+    if (checkExists.length < 1) {

```

Comment:
checkExists might not have updated when this line is executed because it is not within the .then block